roles/check-swlib/tasks/main.yml

```diff
@@ -16,13 +16,27 @@
 - name: check-swlib | Validate base software release
   shell: |
     set -o pipefail
-    {% for i in item.files %}
-      gsutil ls -L gs://{{ swlib_mount_src }}/{{ i }} | \
-        awk '/md5/ { print $3 }'
+    {% for f in item.files %}
```

Comment:
This is getting super complicated to put into embedded, templatized shell;  is there a cleaner way to do this, maybe using ansible loops?  (There might not be;  just asking here)